### PR TITLE
Fix Bug Pictures in search results. Close #239

### LINKED ISSUE #239: picture in search results
we are starting to visualize taxonomy descriptions with pictures. But should the pictures occure in search results: https://de.serlo.org/search?q=stochastik ??

### PR DESCRIPTION
**Pull request by Rahsil**

### CODE REVIEW

File: src/module/Search/src/Search/Provider/TaxonomyProvider.php [hunk 1]
```diff
@@ -77,13 +86,15 @@ public function toDocument($object)
         );
         $keywords   = $normalized->getMetadata()->getKeywords();
         $instance   = $object->getInstance()->getId();
-
+
```
Comment (arekkas) [inline src/module/Search/src/Search/Provider/TaxonomyProvider.php]: remove empty spaces ;)

File: src/module/Search/src/Search/Provider/TaxonomyProvider.php [hunk 2]
```diff
@@ -20,9 +20,17 @@
...
 
 class TaxonomyProvider implements ProviderInterface
 {
+
+    /**
+     * @var StripTags
+     */
+    protected $stripTags;
+
+
```
Comment (arekkas) [inline src/module/Search/src/Search/Provider/TaxonomyProvider.php]: reduce to one empty line

**Comment by arekkas** [comment]
apart from formating issues this looks good to me!

**Comment by arekkas** [comment]
any news @Rahsil ?

**Comment by inyono** [comment]
He had some Git problems today that he tried to fix in the end. Not sure, if he got around those. But should now know that he has to force the push if he changes commits afterwards.

**Comment by arekkas** [comment]
thanks :)

**Pull request merged**